src/PluginStructs/P200_data_struct.h

```diff
@@ -2,6 +2,8 @@
 #define PLUGINSTRUCTS_P200_DATA_STRUCT_H
 
 #include "../../_Plugin_Helper.h"
+#include "src/Helpers/StringConverter.h"
```

Comment:
The path for the include should be relative here.

Reply:
Changed to "../Helpers/StringConverter.h",